sqlite-android/src/main/jni/sqlite/JNIHelp.cpp

```diff
@@ -303,3 +210,8 @@ const char* jniStrError(int errnum, char* buf, size_t buflen) {
     return buf;
 #endif
 }
+
+void* operator new (size_t size)        { return malloc(size); }
+void* operator new [] (size_t size)     { return malloc(size); }
+void operator delete (void* pointer)    { free(pointer); }
+void operator delete [] (void* pointer) { free(pointer); }
```

Comment:
These override C++ memory operators, needed as STL removal perhaps can be put somewhere better.